pkg/relayer/proxy/error_reply.go

```diff
@@ -39,7 +39,9 @@ func (sync *relayMinerHTTPServer) replyWithError(
 	if relayRequest == nil {
 		relayRequest = &types.RelayRequest{
 			Meta: types.RelayRequestMetadata{
-				SessionHeader: &sessiontypes.SessionHeader{},
+				SessionHeader: &sessiontypes.SessionHeader{
+					ServiceId: "unknown_service_id",
```

Comment:
Can you move this into a local constnat somewhere so we can be sure to reuse the same string everywhere?